fs/src/io_uring/memory.rs

```diff
@@ -14,53 +14,25 @@ use {
 // and chunk it in slices of up to 1G each.
 const FIXED_BUFFER_LEN: usize = 1024 * 1024 * 1024;
 
-pub enum LargeBuffer {
-    Vec(Vec<u8>),
-    HugeTable(PageAlignedMemory),
-}
-
-impl Deref for LargeBuffer {
-    type Target = [u8];
-
-    fn deref(&self) -> &Self::Target {
-        match self {
-            Self::Vec(buf) => buf.as_slice(),
-            Self::HugeTable(mem) => mem.deref(),
+/// Allocate memory buffer optimized for io_uring operations, i.e.
+/// using HugeTable when it is available on the host.
+pub fn new_large_buffer(size: usize) -> std::io::Result<PageAlignedMemory> {
+    log::info!("trying to allocate page-aligned buffer of size {}", size);
```

Comment:
degrade this to log::debug and put `size` inside the formatting string (I think this would come up as lint error if you run `./scripts/cargo-clippy.sh`, which you should do before pushing code to a PR)